pymatgen/core/periodic_table.py

```diff
@@ -254,6 +255,18 @@ def __getattr__(self, item: str) -> Any:
             return val
         raise AttributeError(f"Element has no attribute {item}!")
 
+    @property
+    def atomic_orbitals_eV(self) -> dict[str, float]:
```

Comment:
actually, i just noticed this method is not inherited by `Element`, perhaps due to being an `Enum`?